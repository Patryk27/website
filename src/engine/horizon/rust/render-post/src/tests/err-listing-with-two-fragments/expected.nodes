Elem(
    Spanned {
        inner: Elem {
            name: Spanned {
                inner: "listing",
                span: Span {
                    start: 1,
                    end: 8,
                },
            },
            attrs: [],
            children: [
                Text(
                    Spanned {
                        inner: "\n  ",
                        span: Span {
                            start: 9,
                            end: 12,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n        println!(\"hello\");\n    }\n  ",
                        span: Span {
                            start: 16,
                            end: 69,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n\n  ",
                        span: Span {
                            start: 71,
                            end: 75,
                        },
                    },
                ),
                Comment(
                    Spanned {
                        inner: "\n    fn main() {\n        println!(\"world\");\n    }\n  ",
                        span: Span {
                            start: 79,
                            end: 132,
                        },
                    },
                ),
                Text(
                    Spanned {
                        inner: "\n",
                        span: Span {
                            start: 134,
                            end: 135,
                        },
                    },
                ),
            ],
        },
        span: Span {
            start: 0,
            end: 145,
        },
    },
)