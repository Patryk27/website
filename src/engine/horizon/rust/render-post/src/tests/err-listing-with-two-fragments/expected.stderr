[31merror:[0m unexpected node
    â•­â”€[input:8:7]
    â”‚
  8 â”‚ â•­â”€â–¶   <!--
    â”† â”†   
 12 â”‚ â”œâ”€â–¶   -->
    â”‚ â”‚           
    â”‚ â•°â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€â”€ unexpected node
â”€â”€â”€â”€â•¯

compilation failed
